cmd/utils/rotate_key.go

```diff
@@ -0,0 +1,73 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"github.com/frain-dev/convoy/config"
+	"github.com/frain-dev/convoy/database/postgres"
+	"github.com/frain-dev/convoy/internal/pkg/cli"
+	"github.com/frain-dev/convoy/internal/pkg/keys"
+	"github.com/frain-dev/convoy/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+func AddRotateKeyCommand(a *cli.App) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rotate-key <old-key> <new-key>",
+		Short: "Rotates the encryption key by re-encrypting data with a new key",
+		Args:  cobra.ExactArgs(2),
```

Comment:
I see, so this is how it works; my previous comment might be invalid then.

Reply:
Which one though